donate.html

```diff
@@ -2049,91 +2043,93 @@ <h4>Shipping Address</h4>
 {% braintree_js_libs %}
 <script src="/resources/ak_braintree_vzero.js"></script>
 <script>
-  $(function () {
-    {% if not page.custom_fields.donation_only_paypal or page.custom_fields.donation_only_paypal.strip|lower != 'y' %}
-    var form = document.querySelector("#act"),
-      options = {
-        form: form,
-        fields: {
-          number: {
-            selector: '#ak-card_num-hosted'
+  $(document).ready(function() {
+    $(function () {
+      {% if not page.custom_fields.donation_only_paypal or page.custom_fields.donation_only_paypal.strip|lower != 'y' %}
+      var form = document.querySelector("#act"),

```

Comment:
I am not sure if this is the case for vanilla JS but I have two comments here:
1. I often prefer to rely on var vs let or `const` for variables that should not be expected to change e.g the form here. I know with hoisting this is solved but I find more declarative to define constants that way. 
2. Instead of declaring variables using a comma i.e `var x= 2, y =2`, I believe it is clearer to have them on separate lines for readability.

Open to hear your thoughts on this.

Reply:
I actually just wrapped this entire call in a `$(document).ready()` function, because I was periodically seeing some JS errors pop up here - all the code here is preexisting. Definitely agree with your comments though, and this is something that we can perhaps clean up in subsequent PRs!